packages/docs/src/en/directives/teleport.md

```diff
@@ -105,24 +109,26 @@ Alpine does this by looking for event listeners registered on `<template x-telep
 Teleporting is especially helpful if you are trying to nest one modal within another. Alpine makes it simple to do so:
 
 ```alpine
-<div x-data="{ open: false }">
-    <button @click="open = ! open">Toggle Modal</button>
-
-    <template x-teleport="body">
-        <div x-show="open">
-            Modal contents...
-            
-            <div x-data="{ open: false }">
-                <button @click="open = ! open">Toggle Nested Modal</button>
-
-                <template x-teleport="body">
-                    <div x-show="open">
-                        Nested modal contents...
-                    </div>
-                </template>
+<div class="modals">
```

Comment:
At this point it should also be mentioned that if there are more elements with the `modal` class exist in the DOM, only the first one will be targeted.